### PR TITLE
Minecraft data: Minecraft data generater

### PR DESCRIPTION
**Pull request by Defman**
Downloads the Minecraft server jar and uses the built-in generate to obtain useful data. it also extracts useful data from the jar.

### CODE REVIEW

**Comment by Defman** [comment]
I'm not sure about the location of the downloaded jar and generated data.

File: data/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#[cfg(test)]
```
Comment (caelunshun) [inline data/src/lib.rs]: Should probably remove this stub?

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,10 @@
 /target
-/blocks/target
-/server/target
-/core/target
+**/target
+**/target
```
Comment (caelunshun) [inline .gitignore]: I think we only need one `**/target` entry.

**Comment by Defman** [comment]
I have made it such feather-data generates a module for the embedded data as such.
```rust
feather_data::minecraft::lang::EN_US;
feather_data::minecraft::BLOCKS;
feather_data::minecraft::data::loot_tables::entities::BAT;
``

**Comment by caelunshun** [comment]
Thanks, nice work on this.

**Pull request merged**